components/announcement_bar/cloud_trial_announcement_bar/cloud_trial_announcement_bar.tsx

```diff
@@ -205,4 +205,4 @@ class CloudTrialAnnouncementBar extends React.PureComponent<Props> {
     }
 }
 
-export default withGetCloudSubscription(CloudTrialAnnouncementBar);
```

Comment:
I think you should remove the import statement for `withGetCloudSubscription` too.